<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameters</class>
 <widget class="QDialog" name="Parameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>242</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Параметры выходных таблиц</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Параметр</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Занчение</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Emin</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="Emin_cb"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Emax</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="Emax_cb"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>En</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="En_cb"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>nG</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="nG_cb"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>nG_el</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="nG_el_cb"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>nG_ion</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="nG_ion_cb"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>nG_br</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="nG_br_cb"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>nG_br_tet</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QComboBox" name="nG_br_tet_cb"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>nG_photo</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QComboBox" name="nG_photo_cb"/>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>nG_para</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="nG_para_cb"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>nG_annig</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QComboBox" name="nG_annig_cb"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
